content/_ext/pipeline.rb

```diff
@@ -53,8 +53,11 @@
 
   extension Awestruct::IBeams::HandbookExtension.new(:devbook,
                                                      File.expand_path(File.dirname(__FILE__) + '/../doc/developer'))
-
+  
+  # For backwards compatability
```

Comment:
Needs a spelling fix to resolve the typos failure

```suggestion
  # For backwards compatibility
```